core/templates/dev/head/components/embed_modal/embed_exploration_modal_directive.html

```diff
@@ -1,4 +1,18 @@
 <script type="text/ng-template" id="modals/embedExploration">
+  <style>
+    .oppia-embed-modal-code {
+      background-color: #fcfcfc;
+      border: 1px solid #d9d9d9;
+      border-radius: 4px;
+      display: inline-block;
+      padding: 10px;
+    }
+
+    .oppia-embed-modal-code:hover {
+      background-color: #f9f9f9;
+    }
+  </style>
+
   <div class="modal-header">
```

Comment:
Can we wrap the entire thing in a `<div class="oppia-embed-exploration-modal">` tag or similar and then use this as a prefix for the CSS? (So that we have consistency across how we do this for all the files.)